wasm/src/bindings_c2d.cpp

```diff
@@ -279,8 +279,8 @@ class C2DFactory : public Factory {
       // It might be faster to do this on the JS side, and just pass up the arrays...
       // for now, we do it one segment at a time (each turns into an up-call to JS)
       ptr->fillRule(fr);
-      const Vec2D* pts = points.data();
-      for (auto v : verbs) {
+      const Vec2D* pts = path.points().data();
```

Comment:
The sooner we get this in monorepo, the sooner whoever changed the API is responsible for this stuff :)